base/Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM ubuntu:18.04
 
+# Runner user parameters
+ARG UNAME=runner
+ENV UNAME=$UNAME
```

Comment:
About UNAME, I think it should be USER or USERNAME right? Currently this conflicts with [uname](https://man7.org/linux/man-pages/man2/uname.2.html)